cmake/enclave_cmake_wrappers.cmake

```diff
@@ -9,19 +9,18 @@ if (LVI_MITIGATION MATCHES ControlFlow)
     include(apply_lvi_mitigation)
 endif()
 
-# Wrapper of `add_executable`
-macro(add_enclave_executable NAME)
-    add_executable(${NAME} ${ARGN})
-    if (LVI_MITIGATION MATCHES ControlFlow)
+# Wrapper of `add_executable` for adding an enclave with LVI mitigation.
```

Comment:
Consider adding the suggestion that this should only be called from the `add_enclave` function, unlike the other macros.